json/README.md

```diff
@@ -3,9 +3,9 @@
   ~ SPDX-License-Identifier: Apache-2.0
   -->
 
-The new JSON related APIs allows you to convert a JSON String into a binary form that can be stored in a GemFire region. 
+The new JSON related APIs allow you to convert a JSON String into a binary form that can be stored in a GemFire region. 
```

Comment:
change to "...allow you to convert a JSON string into a binary form called JsonDocument that can"...
Also add a sentence saying that "JsonDocuments use less memory and better performance than JSON strings.